fhir-server-test/src/test/java/com/ibm/fhir/server/test/ServerSpecTest.java

```diff
@@ -299,7 +299,7 @@ public void testUpdatePatientVersionAwareError3() throws Exception {
                         .request()
                         .header("If-Match", ifMatchValue)
                         .put(entity, Response.class);
-        assertResponse(response, Response.Status.CONFLICT.getStatusCode());
+        assertResponse(response, Response.Status.PRECONDITION_FAILED.getStatusCode());
```

Comment:
Confirming... this changed due to the IssueType mapping

Reply:
actually, I broke the test on accident, by following the spec instead of the current behavior. 
 from https://www.hl7.org/fhir/http.html#concurrency
> If the version id given in the If-Match header does not match, the server returns a 412 Precondition Failed status code instead of updating the resource.

so i think it was a bug in our current impl and I'm HOPING fixing that doesn't break our touchstone results.

Reply:
P.S. let me know if you think i should open a separate issue for it

Reply:
small... small... suggestion, add the comment to the issue, these are harder to find in the PRs. 

if it's in the issue, at least we can search on it (and I'm suggesting adding to the issue related to this PR only)